modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/test/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokerImplTest.java

```diff
@@ -458,6 +485,42 @@ public void testShortCircuitException() {
 			ddmDataProviderResponse.getStatus());
 	}
 
+	@Test
+	public void testTimeOutChange() {
+		DDMDataProvider ddmDataProvider = mock(DDMDataProvider.class);
+
+		DDMDataProviderRequest.Builder builder =
+			DDMDataProviderRequest.Builder.newBuilder();
+
+		DDMDataProviderRequest ddmDataProviderRequest = builder.build();
+
+		int timeout1 = 10000;
+
+		DDMDataProviderInvokeCommand command1 =
+			new DDMDataProviderInvokeCommand(
+				"ddmDataProviderInstanceName", ddmDataProvider,
+				ddmDataProviderRequest,
+				_createDDMRESTDataProviderSettingsWithTimeout(timeout1));
+
+		int executionTimeoutInMilliseconds1 =
+			_getExecutionTimeoutInMilliseconds(command1);
+
+		Assert.assertEquals(timeout1, executionTimeoutInMilliseconds1);
+
+		int timeout2 = 15000;
+
+		DDMDataProviderInvokeCommand command2 =
+			new DDMDataProviderInvokeCommand(
+				"ddmDataProviderInstanceName", ddmDataProvider,
+				ddmDataProviderRequest,
+				_createDDMRESTDataProviderSettingsWithTimeout(timeout2));
+
+		int executionTimeoutInMilliseconds2 =
+			_getExecutionTimeoutInMilliseconds(command2);
```

Comment:
@cleydyr, do you think we can reuse the variables `timeout1` and `executionTimeoutInMilliseconds1` here? If so, can we rename them to `timeout` and `executionTimeoutInMilliseconds`, respectively?

Reply:
Yes we can. I was just half-way to refactor the two very similar parts, so I used two of them. :smile: